localstack/services/cloudformation/models/apigateway.py

```diff
@@ -877,6 +877,21 @@ def _handle_result(
             resource: dict,
         ):
             resource["PhysicalResourceId"] = result["domainName"]
+            hosted_zones = connect_to().route53.list_hosted_zones()
+            """
+            The hardcoded value is the only one that should be returned but due limitations it is not possible to
+            use it.
+            """
```

Comment:
```suggestion
            # The hardcoded value is the only one that should be returned but due limitations it is not possible to use it.
```

Reply:
what limitations?

Reply:
The CloudfrontDistributionZoneId that it's hardcoded it's the only value that should be return. But it requires that the cloudfront service always have that existing zone. which it doesn't. so the resource model returns the id of the first zone it finds before putting the correct value.